src/hex_api_release.erl

```diff
@@ -14,9 +14,9 @@
 
 -type publish_params() :: [{replace, boolean()}].
 
--type retirement_reason() :: other | invalid | security | deprecated | renamed.
+-type retirement_reason() :: binary().
```

Comment:
we don't use this type anymore anywhere, can you remove it while at it?

Reply:
Sure.

Reply:
28145ef